.github/workflows/publish.yml

```diff
@@ -229,6 +229,11 @@ jobs:
       - name: List artifacts
         run: ls -R artifacts
         shell: bash
+      - name: Trigger Docs Release
+        run: |
+          git rebase $GITHUB_REF_NAME latest
+          git push
+          git checkout -
```

Comment:
Is $GITHUB_REF_NAME the current commit?
Will `git checkout -` correctly return to the previous commit?

Reply:
GITHUB_REF_NAME Is trigger dependent - in our case here it is the tag of the version number eg 16.10.0

Reply:
The branch update should come last, we only want to do it if publishing succeeds. We also don’t want to do it if it is a prerelease right? You will need to add detection for that

Reply:
You will need a script to inspect the tag version number, you can see I have an existing isPrerelease function in changelog.ts you can copy from

Reply:
In typescript-eslint I did a force recreation and push rather than a rebase and I think it’s preferable:

git branch -f latest
git push -f origin latest

We always want the latest branch to reflect the latest changes for the particular version, and we don’t care about the git history and we then don’t have any risk of conflicts causing the process to fail which you would with a rebase

Reply:
Force recreation and push is how I handle it for nx-dotnet's docs too, works pretty well

Reply:
It appears that github provides `github.event.release.prerelease` as a differentiator between release and pre-release versions.